CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # CHANGELOG
 
+* `3.2.6` Release - [3.2.500](#326)
```

Comment:
should't be 3.2.600 instead ?

Reply:
Yes should be `3.2.6` actually , good catch